src/components/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+  OverviewPage,
+  DetailsPage,
+  DetailsItem,
+  k8sGetResource,
+  K8sModelCommon /* , useK8sWatchResource */,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import type { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { useParams } from 'react-router-dom';
+
+const WorkspaceDetailsTabs = (workspace: K8sResourceCommon, loaded: boolean, error: { message: string; status: number }) => {
+  return [
+    {
+      key: 'overview',
+      title: 'Overview',
+      content: (
+        <OverviewPage loaded={loaded} resource={workspace} loadError={error} rightColumn={<></>}>
+          <DetailsItem label="Services" resource={workspace} path="spec.type.name" loaded={loaded} />
+        </OverviewPage>
+      ),
+      ariaLabel: 'Overview',
+    },
+  ];
+};
+
+const WorkspaceDetailsBreadcrumbs = (workspaceName: string) => [
+  { name: 'Workspaces', path: '/workspaces' },
+  { name: 'Workspace details', path: workspaceName ? `/workspaces/${workspaceName}` : '' },
```

Comment:
```suggestion
  { name: 'Workspace Details', path: workspaceName ? `/workspaces/${workspaceName}` : '' },
```
if https://www.patternfly.org/v4/ux-writing/capitalization#capitalization-across-red-hat-uis is what applies here.  Is it just me, or do these rules keep changing?

Reply:
Good question.  There are multiple designs out there and see what they have.

Reply:
It is all over the place - so let's just change it to lower case to match the patternfly doc.